Automattic/Edit-Flow
# Remove jquery.quicksearch.js file

Fix #661 

## Description

The reason for this PR is mentioned in #661. 

What I've done to do this cleanup: 

- Searched `jquery.quicksearch.js` and removed the relevant places. 
- Searched `jquery-quicksearch` and removed the relevant things. 
- Searched `quicksearch` and removed the relevant things. After this PR, there are still few places for this keyword `quicksearch` in  modules/notifications/lib/notifications.css and modules/user-groups/lib/user-groups.css but I think we should keep it. 
 
## Steps to Test

- Apply this PR.
- Apply the diff in the `Note` section in this PR https://github.com/Automattic/Edit-Flow/pull/649
- Open browser console and watch for errors. 
- Check across all Edit Flow places as mentioned in PR 649. 
- Focus on two places that are mostly affected by this PR: (1) Notifications section when editing a post, and (2) WP Admin > Edit Flow > User Groups > Edit a group, where user groups display.
